### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by jensens**
- pyupgrade/manual removal of py2 leftovers.
- isort/black
- depend on plone.base and remove implicit circular dependency on CMFPlone

### CODE REVIEW

**Comment by jensens** [comment]
ops

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
Plone 5 is on its own branch now, so no tests are running there.

**Comment by gforcada** [comment]
@jensens could you please either use `plone/meta` or ask for someone to do that? otherwise running this home made cleanups without adding the pre-commit/tox configuration makes the changes not being enforced at all 😕 

I can try to keep up with your peace, but otherwise opening a generic issue on plone/meta mentioning all the packages that you have manually cleaned up would be great.

Specially since enabling pre-commit.ci and enabling GHA is something that happens outside each code/repository but on the organization level.

File: plone/app/textfield/browser.py [hunk 1]
```diff
@@ -43,7 +42,7 @@ def __call__(self, value=None, fieldName=None, mimeType=None):
             if not self.major or not self.minor:
                 mimeType = value.outputMimeType
             else:
-                mimeType = "%s/%s" % (self.major, self.minor,)
+                mimeType = "{}/{}".format(self.major, self.minor)
```
Comment (gforcada) [inline plone/app/textfield/browser.py:45]: Running `pyupgrade --py38-plus` would turn this into a nice f-string 😄

**Review by gforcada — APPROVED** [review]

File: plone/app/textfield/browser.py (hunk 1 above)
Comment (jensens) [reply plone/app/textfield/browser.py:45]: Well, this is what I did: `find . -name "*.py" -exec pyupgrade --py38-plus {} +`.
Anyway, running it again turned it into an f-string.

Comment (mauritsvanrees) [reply plone/app/textfield/browser.py]: I see that too: sometimes running pyupgrade twice still makes changes in the second run.

Comment (gforcada) [reply plone/app/textfield/browser.py]: Maybe it's `pyupgrade` + `black` + `pyupgrade` ? 🤔 oh, well, `pre-commit` will warn us 🍀

**Pull request merged**